### DISCUSSION TITLE
PySE - valor do auxílio da PSF

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#138] [Pergunta]
A Python Sudeste solicitou auxílio da Python Software Foundation e este foi atendido.
Gostaríamos de saber o valor disponível para utilização pós conversão/taxa da APyB para fecharmos o caixa do evento e escrevermos o formulário de Report.

**Comment by cecivieira** [comment]
@anadulce o detalhamento das movimentações até junho podem ser encontradas aqui:
https://financeiro.python.org.br/apyb/account/Assets:Eventos:Pyse2024/

quanto a jul e agosto, estão em fase de revisão, mas já podem ser encontradas aqui: https://github.com/apyb/financeiro/pulls

O valor final fica ~R$17.500
